resfit/fit_resonator/fit_S_data.py

```diff
@@ -1016,6 +1016,7 @@ def fit_resonator(filename: str,Method,normalize: int,dir: str = None, data_arra
     #make a folder to put all output in
     result = time.localtime(time.time())
     output = str(result.tm_year)
+    output = str(Method.method) + '_' + output
```

Comment:
why is output being overwritten here? Did you mean to delete the previous line?

Reply:
It's adding to the previous output, not overwriting it. This was already in the code, I just changed the format of the name before "+ '_' + output". I can change it to:
 output = str(Method.method) + '_' + str(result.tm_year)
so it looks cleaner.

Reply:
Ah that makes sense.